version_test.go

```diff
@@ -0,0 +1,14 @@
+package specs
+
+import (
+	"regexp"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+	re := regexp.MustCompile(`\d+\.\d+\.\d+`)
```

Comment:
SemVer 2.0 also allows for pre-release versions and build metadata.  See [sections 9 and 10][1].  Surely someone else has already written a SemVer validator in Go?

[1]: http://semver.org/spec/v2.0.0.html